packages/webapp/pages/squads/[handle]/[token].tsx

```diff
@@ -128,21 +126,14 @@ const SquadReferral = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [member, trackedImpression]);
 
-  const sourceId = member?.source?.id;
   const { mutateAsync: onJoinSquad } = useMutation(
-    () => joinSquadInvitation({ sourceId, token }),
+    useJoinSquad({
+      squad: { handle, id: member?.source?.id },
+      referralToken: token,
```

Comment:
The token here is either a user token or the squad token?
Just for my understanding.

Reply:
Here it is always user token, with open squad we don't send token at all.

Referral is attributed through new referral cookie.